components/shared/IdentityPopover.vue

```diff
@@ -103,39 +104,70 @@ export default class IdentityPopover extends mixins(
     await this.fetchNFTStats()
   }
 
+  @Watch('identity.address')
   protected async fetchNFTStats() {
     try {
-      const query =
-        this.urlPrefix === 'rmrk'
-          ? await import('@/queries/nftStatsByIssuer.graphql')
-          : await import('@/queries/unique/nftStatsByIssuer.graphql')
-      this.$apollo.addSmartQuery('collections', {
-        query: query.default,
-        manual: true,
-        client: this.urlPrefix,
-        loadingKey: 'isLoading',
-        result: this.handleResult,
-        variables: () => {
-          return {
+      const data = this.$store.getters['identityMint/getIdentityMintFor'](
+        this.identity.address
+      )
+      if (data && Date.now() - data.updatedAt < 12 * 60 * 60 * 1000) {
```

Comment:
from date fns

```suggestion
      if (isBefore(data.updatedAt, subHours(Date.now, 12))) {
```

Reply:
Already change, u really know much🙂
`const data = this.$store.getters['identityMint/getIdentityMintFor'](
        this.identity.address
      )
      if (
        data?.updatedAt &&
        isAfter(data.updatedAt, subHours(Date.now(), 12))
      ) {
        // if cache exist and within 12h
        await this.handleResult({ data, type: 'cache' })
      } else {
        const query =
          this.urlPrefix === 'rmrk'
            ? await import('@/queries/nftStatsByIssuer.graphql')
            : await import('@/queries/unique/nftStatsByIssuer.graphql')`

Reply:
> Already change, u really know much🙂 
` if ( data?.updatedAt && isAfter(data.updatedAt, subHours(Date.now(), 12)) ) 
{ // if cache exist and within 12h await this.handleResult({ data, type: 'cache' }) } 
else { ...
`